packages/service/src/domain/commands/processors/playback-current-command.ts

```diff
@@ -0,0 +1,73 @@
+import { Command } from '@service/domain/commands/model/command';
+import { CommandProcessingResponse, CommandProcessingResponseBuilder } from '@service/domain/commands/model/command-processing-response';
+import { CommandParameters, CommandParametersBuilder, CommandProcessor } from '@service/domain/commands/model/command-processor';
+import { RegisterCommand } from '@service/domain/commands/registry/register-command';
+import { PlayerStateRequestEvent } from '@service/event-stream/model/events';
+import { PlayerEventStream } from '@service/event-stream/player-event-stream';
+import { PlayerState } from '@service/domain/player-state/player-state';
+import { Service } from 'typedi';
+import { Song } from '@service/lib';
+
+@RegisterCommand
+@Service()
+class PlaybackCurrentCommand extends CommandProcessor {
+  constructor(private playerEventStream: PlayerEventStream) {
+    super();
+  }
+
+  async execute(command: Command): Promise<CommandProcessingResponse> {
+    const includeQueue = command.rawArgs === 'queue';
+
+    const playerStateRequest: PlayerStateRequestEvent = { id: 'PlayerStateRequestEvent' };
+    const currentPlayerState: PlayerState = await this.playerEventStream.sendToPrimaryPlayer(playerStateRequest);
+
+    const text = this.buildMessage(
+      currentPlayerState.currentlyPlaying?.song ?? null,
+      currentPlayerState.queue ?? [],
+      includeQueue,
+    );
+
+    return new CommandProcessingResponseBuilder()
+      .fromMarkdown(text)
+      .build();
+  }
+
+  private buildMessage(current: (Song | null), queue: Song[], includeQueue: Boolean): string {
+    const queueSongsTitles = includeQueue ? queue.map((song) => `  - ${song.name.truncated(50, false)}`) : [];
```

Comment:
| `useWordBoundary: true`                                                                                                                                                                                                                                                                                                                              	| `useWordBoundary: false`                                                                                                                                                                                                                                                                                                   	|
|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|
| <pre><br>Obecnie gramy:<br>  a-ha - Take On Me (Official Video) [Remastered in 4K]<br>W kolejce:<br>  - New Order - Thieves Like Us<br>  - a-ha - Take On Me (Official Video) [Remastered…<br>  - M - Pop Muzik (Official Video)<br>  - New Order - Bizarre Love Triangle (Official…<br>  - New Order - True Faith (1987) (Official Music…<br></pre> 	| <pre><br>Obecnie gramy:<br>  Heaven 17 - Temptation (Official Video)<br>W kolejce:<br>  - Mad World by Tears For Fears Original HQ 1983<br>  - Prince - 1999 (Official Music Video)<br>  - Tears For Fears - Pale Shelter<br>  - Depeche Mode - Just Can't Get Enough (Official Vi…<br>  - KRAFTWERK - THE MODEL<br></pre> 	|